src/java/main/org/apache/zookeeper/server/quorum/Observer.java

```diff
@@ -125,6 +125,8 @@ protected void processPacket(QuorumPacket qp) throws IOException{
             ObserverZooKeeperServer obs = (ObserverZooKeeperServer)zk;
             obs.commitRequest(request);            
             break;
+        default:
+            LOG.error("Invalid packet type received by Observer: " + qp.getType());
```

Comment:
Can we change log message like below to improve readability.

`LOG.error("Invalid packet type: {} received by Observer", qp.getType());`